irs-policy-store/src/main/java/org/eclipse/tractusx/irs/policystore/controllers/PolicyStoreController.java

```diff
@@ -102,6 +102,8 @@ public class PolicyStoreController {
     public static final int DEFAULT_PAGE_SIZE = 10;
     public static final int MAX_PAGE_SIZE = 1000;
     public static final String SEARCH = "search";
+    public static final String POLICY_API_TAG = "Item Relationship Service - Policy API";
```

Comment:
Since this is what Swagger UI uses to group endpoints, "Item Relationship Service - Policy API" is a bit long and perhaps redundant.
Maybe just call it "Policy Store API"